valohai_yaml/schema/file-item.yaml

```diff
@@ -0,0 +1,16 @@
+---
+type: object
+required:
+- name
+- path
+additionalProperties: false
+properties:
+  name:
+    type: string
+    description: The symbolic name of this file.
+  description:
+    type: string
+    description: The description of the file.
+  path:
+    type: string
+    description: The path of the file.
```

Comment:
This is actually the relative path to the file from the endpoint's working directory. An example or two in the `description` might help, e.g. just adding "e.g. `data/serialized_model.hdf5`".